layout-server/src/main/java/org/finos/vuu/layoutserver/model/ApplicationLayout.java

```diff
@@ -0,0 +1,31 @@
+package org.finos.vuu.layoutserver.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Data
+@Entity
+@RequiredArgsConstructor
+@AllArgsConstructor
+public class ApplicationLayout {
+    @Id
+    private String username;
+
+    @Column(columnDefinition = "JSON")
+    private String definition;
+
+    public String extractDefinition() {
+        String extractedDefinition = definition;
+
+        if (extractedDefinition.startsWith("\"") && extractedDefinition.endsWith("\"")) {
+            extractedDefinition = extractedDefinition.substring(1, extractedDefinition.length() - 1);
+        }
+
+        return extractedDefinition.replaceAll("\\\\", "");
+    }
```

Comment:
I don't like having to do this. Any suggestions for a better way would be greatly appreciated.

Reply:
I'm still having to do the same string manipulation, but it's now tidied into the `JsonNodeConverter` class.